### PR TITLE
140 sub remove hdr memmap

### PR DESCRIPTION
**Pull request by dfsp-spirit**
A step towards #140

Author Guidelines
-----------------
- [x] Is the change set **< 400 lines**?
```python
    +210 - 826 < 400
    True
```

- [x] Was the code checked for memory leaks/performance bottlenecks? (n.z.)
- [x] Is the code running locally **and** on the ESI cluster?
- [x] Is the code running on all supported platforms?

Reviewer Checklist
------------------
- [ ] Are testing routines present?
- [ ] Do **parallel** loops have a set length and correct termination conditions?
- [ ] Do objects in the global package namespace perform proper parsing of their input? 
- [ ] Do code-blocks provide novel functionality, i.e., no re-factoring using builtin/external packages possible?
- [ ] Code layout
  - [ ] Is the code PEP8 compliant?
  - [ ] Does the code adhere to agreed-upon naming conventions?
  - [ ] Are keywords clearly named and easy to understand?
  - [ ] No commented-out code?
- [ ] Are all docstrings complete and accurate?

### CODE REVIEW

**Comment by dfsp-spirit** [comment]
It's green! But how can I upload the coverage data?

**Comment by tensionhead** [comment]
Mmh, I guess because the base branch `140-multiple-outputs` has no coverage report/data, so I guess it's fine for this sub-branch! When we merge into dev, the coverage will come back..

File: syncopy/datatype/continuous_data.py [hunk 1]
```diff
@@ -183,38 +183,25 @@ def time(self):
 
     # # Helper function that reads a single trial into memory
     # @staticmethod
-    # def _copy_trial(trialno, filename, dimord, sampleinfo, hdr):
+    # def _copy_trial(trialno, filename, dimord, sampleinfo):
```
Comment (tensionhead) [inline syncopy/datatype/continuous_data.py:186]: Looks all nice and clean, just wondering if we need that commented-in function at all..

**Review by tensionhead — APPROVED** [review]
Great job, good to finally remove this unused functionality.. especially the CR got rid of some nested if/else :+1:

**Comment by tensionhead** [comment]
Also closes #42 !

File: syncopy/datatype/continuous_data.py (hunk 1 above)
Comment (dfsp-spirit) [reply syncopy/datatype/continuous_data.py:186]: most likely not. we can remove it in the outer branch, I guess.

**Pull request merged**